docs/help-center/using-famedly/qr-verification.mdx

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: QR-Verification
+title: QR-Verification
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 5
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">QR-Verification</h2>
+    <p class="hero__subtitle">How to use QR-verification and what to do if your device can't scan the code.</p>
+    <p>With the QR-verification feature you can easily verify new devices by scanning a QR-code.
+    </p>
+  </div>
+</div>
+
+## Usage of Verification
+
+You will get redirected to the verification page automatically in the login-flow, 
+if at least one of the devices you want to use for verification is a mobile device.  
+If you nevertheless get redirected to the number- or emoji-verification instead, 
+please check if there is an update for one of your devices. 
+
+<Tabs>
+  <TabItem value="web" label="Web" default>
+  <ol>
+    <li>Accept the verification request by clicking on <b>Continue</b> in the <b>New verification request</b> dialog.</li>
+    <li>If the other device is a mobile device, the screen on your web tab will display a QR code. If it's another web device, it will fallback to number verification.</li>
+    <li>Scan the QR code displayed on your web browser with your mobile device. If both devices are in web browsers, compare the numbers and confirm them.</li>
+    <li>If by some reason you are not able to scan the QR code, you can tap on <b>Compare numbers instead of QR-Code</b>.</li>
+    <li>After scanning the QR code, you will be asked if it was really you who scanned the QR code on your already active device. Click on <b>Yes, it was me</b> to complete the verification.</li>
+  </ol>
+  </TabItem>
+  <TabItem value="mobile" label="Mobile">
+  <ol>
+    <li>Accept the verification request by tapping on <b>Continue</b> in the <b>New verification request</b> dialog.</li>
+    <li>The already active device will show a QR code and the device you want to verify will open a QR code scanner.</li>
```

Comment:
> device you want to verify will open a QR code scanner

sounds wrong, the QR code scanner will open where scanning is supported (usually mobile devices) (if both devices support scanning, f.ex 2 mobiles we automatically choose one device to scan and the other to show.)

Reply:
@techno-disaster I tested it before and every time when both devices could scan, the scanner opened on the device that I try to verify so I wrote it that way.
So what exactly do you want me to change?

Reply:
This is the logic for choosing which device scans and which device shows https://github.com/famedly/app/blob/main/lib/dialogs/key_verification_dialog.dart#L443, so if there were 2 mobiles, whichever deviceID is first shows, the next one scans

Reply:
@techno-disaster sorry I didn't come back to this before:
So the code means technically both devices could be asked to scan because of the `entries.sort()` call. So in my case my Entry just was first all the times after sorting.
But why not directly say if both devices can scan, the requesting device will? That would save code and would be way more logic to me and better to document.